com.woltlab.wcf/templates/__totpNewDeviceContainer.tpl

```diff
@@ -0,0 +1,44 @@
+<section id="{@$container->getPrefixedId()}Container"{*
+	*}{if !$container->getClasses()|empty} class="{implode from=$container->getClasses() item='class' glue=' '}{$class}{/implode}" {/if}{*
+	*}{foreach from=$container->getAttributes() key='attributeName' item='attributeValue'} {$attributeName}="{$attributeValue}"{/foreach}{*
+	*}{if !$container->checkDependencies()} style="display: none;"{/if}{*
+*}>
+	{if $container->getLabel() !== null}
+		{if $container->getDescription() !== null}
+			<header class="sectionHeader">
+				<h2 class="sectionTitle">{@$container->getLabel()}{if $container->markAsRequired()} <span class="formFieldRequired">*</span>{/if}</h2>
+				<p class="sectionDescription">{@$container->getDescription()}</p>
+			</header>
+		{else}
+			<h2 class="sectionTitle">{@$container->getLabel()}{if $container->markAsRequired()} <span class="formFieldRequired">*</span>{/if}</h2>
+		{/if}
+	{/if}
+	
+	<div class="multifactorTotpNewDevice">
+		{if $container->getNodeById('secret')->isAvailable()}
+			{@$container->getNodeById('secret')->getHtml()}
+		{/if}
+		
+		<div class="multifactorTotpNewDeviceFields">
+			{foreach from=$container item='child'}
+				{if $child->getId() !== 'secret' && $child->getId() !== 'submitButton' && $child->isAvailable()}
+					{@$child->getHtml()}
+				{/if}
+			{/foreach}
+			
+			<div class="formSubmit">
```

Comment:
Is the `.formSubmit` element always available on purpose?

Reply:
No, those checks should be there.

Reply:
Okay. Would you recommend moving the `<div>` into the `{if $container->getNodeById('submitButton')->isAvailable()}` then?

Reply:
Yes.